src/syft/lib/__init__.py

```diff
@@ -90,11 +91,13 @@ def create_lib_ast(client: Optional[Any] = None) -> Globals:
     torch_ast = create_torch_ast(client=client)
     torchvision_ast = create_torchvision_ast(client=client)
     # numpy_ast = create_numpy_ast()
+    plan_ast = create_plan_ast(client=client)
 
     lib_ast = Globals(client=client)
     lib_ast.add_attr(attr_name="syft", attr=python_ast.attrs["syft"])
     lib_ast.add_attr(attr_name="torch", attr=torch_ast.attrs["torch"])
     lib_ast.add_attr(attr_name="torchvision", attr=torchvision_ast.attrs["torchvision"])
+    lib_ast.syft.add_attr("core", attr=plan_ast.syft.core)
```

Comment:
@madhavajay - I don't know if adding this here is violating a design principle. Please deeply consider whether exposure to core like this is something we want. 

I *think* it's ok but I've been away from the code for a bit so I'll defer to your expertise.

Reply:
Do we need to be able to create Plans remotely?
```
remote_plan = alice_client.syft.core.node.common.plan.plan.Plan([a1,a2, a3, a4, a5, a6, a7, a8])
```

Reply:
Maybe, but thats not why I added it. If I remove it I get
ValueError: Path core.node.common.plan.plan.Plan.execute not present in the AST.
on `plan.execute()`

Reply:
Discussed with @koenvanderveen and for now I think theres no down side since its identical to manually executing all of the same functionality remotely, and has the added benefit of possible use cases.